### ISSUE TITLE
podman logs should print container names instead of ids

### ISSUE THREAD

**Issue by tanwald** [containers/libpod#4962]
/kind feature

**Description**

As soon as you print the logs of more than one container you can hardly read the output. (Especially in combination with  #4961) 

**Steps to reproduce the issue:**

1. Run `podman logs <some-container> <another-container> ...`

**Describe the results you received:**
```
aac136ff9cc9 11:39:24,169 INFO  [org.jboss.modules] (main) JBoss Modules version 1.9.0.Final
ac0e1a1880d3 11:39:45,141 INFO  [org.jboss.modules] (main) JBoss Modules version 1.9.0.Final
```

**Describe the results you expected:**

```
some-container    11:39:24,169 INFO  [org.jboss.modules] (main) JBoss Modules version 1.9.0.Final
another-container 11:39:45,141 INFO  [org.jboss.modules] (main) JBoss Modules version 1.9.0.Final
```
An option to print logs of different containers in different colours would help even more [see docker-compose].

**Comment by baude** [comment]
@rhatdan @mheon @haircommander  What do you guys think? We could also just a --names option.

**Comment by mheon** [comment]
Option is best way to handle this, I think.

**Comment by Akasurde** [comment]
@mheon @baude If you don't mind, I would like to take this issue. Thanks.

**Comment by rhatdan** [comment]
It is all yours @Akasurde

**Comment by baude** [comment]
@Akasurde thanks!

**Comment by rhatdan** [comment]
@Akasurde Any progress?

**Comment by Akasurde** [comment]
@rhatdan I have an initial version of the fix in #5244. Could you please review? Thanks.
